runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -211,7 +211,9 @@
     @Getter
     private static final String mpObj = mp + "object.";
     @Getter
-    private static final MetricRegistry metrics = new MetricRegistry();
+    private static MetricRegistry defaultMetrics = new MetricRegistry();
```

Comment:
We tried that. But then the address space view requires the metrics registry which it obtains statically. If we make it an object parameter, we would need to pass the registry to the address space view too on creation. Would that be fine?